afpy/templates/rest/rss.rst

```diff
@@ -9,8 +9,8 @@ Flux de l'AFPy
 Vous pouvez suivre les informations de l'AFPy en vous abonnant aux flux
 suivants :
 
-- `Actualités </feed/actualites/rss.xml>`_
-- `Offres d'emploi </feed/emplois/rss.xml>`_
+- `Actualités </feed/news/rss.xml>`_
+- `Offres d'emploi </feed/jobs/rss.xml>`_
```

Comment:
Si on change le chemin du rss.xml, on va casser tous les abonnements, est-ce nécessaire de changer ?

Reply:
Fixed in de310e8a1536963473a857e2161d4a15c18c2271